test/simulation/config.go

```diff
@@ -4,15 +4,15 @@ import "time"
 
 const (
 	defaultXudKill    = true
-	defaultXudCleanup = true
+	defaultXudCleanup = false
```

Comment:
Did you mean to flip this?

With https://github.com/ExchangeUnion/xud/pull/923/commits/631d60792913de72e26b8e3feb3f63e6f43cb9d1 I have been using `false` here locally as well and I can run the sim tests without having to manually delete the data dir folders each time, so maybe it would make sense to leave it `false` here, but I wanted to make sure it was intentional.

Reply:
The reason was only for debugging the certificate generation issue (`travis-debug` commits), so i'm going to remove this.

I'd say not to prefer using memory db, because there's no problem with deleting these folders, and better to have the behavior close as possible to production-like. Also, I sometimes read this database after runs to inspect various issues. But i'll review your solution better and comment there.